bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/candidate/CandidateObsComparator.java

```diff
@@ -34,7 +34,7 @@ public int compare(Obs o1, Obs o2) {
 			return direction * ((ret != 0) ? ret : -o1.compareTo(o2));
 		}
 		
-		return direction * attr.getComparator().compare(o1, o2);
+		return direction * attr.getComparator(variant.getSchedule().getMiddlePoint()).compare(o1, o2);
```

Comment:
I'm not sure, but the midpoint probably makes sense here. We're sorting observations based on some attribute which might vary based on time.  But to compare one observation to another we have to fix the time to be used.  The midpoint of the night seems as good as any time in this case?  @tpolecat do you have any input?  I'm not sure how this is used.